docs/UserGuide.md

```diff
@@ -155,18 +155,42 @@ Examples:
 
 Loans a book to some user, which has a due date.
 
-Format: `loan USER_INDEX BOOK_INDEX [DUE DATE]`
+Format: `loan USER_INDEX BOOK_INDEX [DUE_DATE]`
 
 * Loans the book to some user at their respective specified `INDEXES`.
 * The indexes refer to the index number shown in the displayed user and book list respectively.
 * The indexes **must be a positive integer** 1, 2, 3, …​
 * The respective specified `INDEXES` **must be present in their lists**.
-* The books that are loaned out will appear at the top of the book list.
+* The books that are loaned out will appear at the top of the book list after closing and reopening of app.
```

Comment:
'after closing and reopening of app' --> 'after relaunching the app' may be better

Reply:
Fixed